docs/usage/configuration-options.md

```diff
@@ -753,7 +753,7 @@ image: my.old.registry/aRepository/andImage:1.18-alpine
   "customManagers": [
     {
       "customType": "regex",
-      "fileMatch": ["values.yaml$"],
+      "filePatterns": ["values.yaml"],
```

Comment:
wait, it's not the same
```suggestion
      "filePatterns": ["/values.yaml$/"],
```

Reply:
yeah, I had thought of that ..missed during review :)